sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -214,14 +408,111 @@ export interface SessionReceiverOptions {
     sessionId?: string;
 }
 
+// @public
+export type SqlParameter = {
+    key: string;
+    value: string | number;
+    type: string;
+};
+
+// @public
+export type SqlRuleAction = SqlRuleFilter;
+
+// @public
+export interface SqlRuleFilter {
+    compatibilityLevel?: number;
+    requiresPreprocessing?: boolean;
+    sqlExpression?: string;
+    sqlParameters?: SqlParameter[];
```

Comment:
`SqlParameter` to `SqlRuleParameter` ?

Reply:
No, the parameter is for the sql expression :)